backends/dpdk/dpdkContext.cpp

```diff
@@ -109,6 +109,45 @@ void DpdkContextGenerator::CollectTablesAndSetAttributes() {
         }
     }
 
+    for (auto ed :  structure->externDecls) {
+        if (auto type = ed->type->to<IR::Type_Specialized>()) {
+            auto externTypeName = type->baseType->path->name.name;
+            if (externTypeName == "Counter" ||
+                externTypeName == "Register" ||
+                externTypeName == "Meter" ||
+                externTypeName == "Hash" ||
+                externTypeName == "InternetCheckSum") {
+                struct externAttributes externAttr;
+                externAttr.externalName = ed->controlPlaneName();
+                externAttr.externType = externTypeName;
+                if (externTypeName == "Counter") {
+                    if (ed->arguments->size() != 2) {
+                        ::error(ErrorType::ERR_EXPECTED,
+                                "%1%: expected number of counters and type of counter"
```

Comment:
Isn't it unexpected number of arguments? There is no mention of number of arguments expected in the error message.

Reply:
Tried to give little more detail about the expected argument

Reply:
Oh you mean the error type?

Reply:
Updated.